# HG changeset patch
# User Carlo de Falco <carlo.defalco@polimi.it>
# Date 1450188647 -3600
#      Tue Dec 15 15:10:47 2015 +0100
# Node ID 4d14b0a22b298847a575e2fa2ec79a6ae18815c2
# Parent  59d49077825ffda53ff3b43d86e480dfe81f8894
fix seealso in odeplot
* scripts/ode/odeplot.m: remove odepkg from seealso list
  in docstring.

diff --git a/scripts/ode/odeplot.m b/scripts/ode/odeplot.m
--- a/scripts/ode/odeplot.m
+++ b/scripts/ode/odeplot.m
@@ -37,17 +37,17 @@
 %# @example
 %# fvdb = @@(t,y) [y(2); (1 - y(1)^2) * y(2) - y(1)];
 %# 
 %# opt = odeset ("OutputFcn", @@odeplot, "RelTol", 1e-6);
 %# sol = ode45 (fvdb, [0 20], [2 0], opt);
 %# @end example
 %# @end deftypefn
 %#
-%# @seealso{odepkg}
+%# @seealso{odeset,odeget}
 
 function [varargout] = odeplot (t, y, flag, varargin)
 
   ## No input argument check is done for a higher processing speed
   persistent fig; persistent told; 
   persistent yold; persistent counter;
 
   if (strcmp (flag, "init")) 
